production/Handler.py

```diff
@@ -22,30 +22,67 @@ def __init__(self, orderQueue):
         self.orders = orderQueue
         self.uploadOrders = Queue()
         self.queue = Queue()
-        self.imageList = self.getDirectoryList(self.directoryName)
+        self.listFileName = Utility.UPLOADS_FILE_NAME
+        self.exitMessage = ""
+        
+#     def run(self):
```

Comment:
Old version of run. Maintaining for posterity currently, will eventually remove these comments.